communications/newsletters/newsletter_02_Oct2019.md

```diff
@@ -0,0 +1,74 @@
+
+We ran **4 Online Collaboration Cafes** ([videos](https://www.youtube.com/channel/UCPDxZv5BMzAw0mPobCbMNuA)) and partnered with the Mozilla Open Leader X - [Open Life Science mentoring program](https://openlifesci.org).
+
+Shout out to Sarah Gibson and Rachael Ainsworth who gave talks and ran a workshop on The Turing Way Project. Thanks to Paula Andrea Martinez and Philip Durbin for engaging with the project from Australia and USA. 
+
+We are excited to meet many of you at [MozFest](https://www.mozillafestival.org/en/) in London, and have been thinking of ways to involve people in different time zones.
+
+Find more details on these topics below. :point_down:
+
+See you after Halloween! :jack_o_lantern: :maple_leaf:
+
+## Community meetings
+
+### Next Collaboration Cafes
+
+This month’s collaboration cafes will take place on 2 October at 15:00 BST ([your local time](https://arewemeetingyet.com/London/2019-10-02/15:00/TuringWay-CollaborationCafe)), 16 October at 19:00 BST ([your local time](https://arewemeetingyet.com/London/2019-10-16/19:00/TuringWay-CollaborationCafe)), and 30 October at 08:00 BST ([your local time](https://arewemeetingyet.com/London/2019-10-30/08:00/TuringWay-CollaborationCafe)). We would love to have you on these calls to continue working together on different GitHub issues or help you get started if you are new to the project. You can indicate your participation by signing up on this HackMD: [https://hackmd.io/@KirstieJane/CollabCafe](https://hackmd.io/@KirstieJane/CollabCafe). 
+
+### MozFest 2019
+
+Several community members of The Turing Way will be attending [Mozilla Festival 2019](https://www.mozillafestival.org/en/) in London from 21 to 27 October. If you are in town and also attending this conference, get in touch through our [Gitter channel](https://gitter.im/alan-turing-institute/the-turing-way) or other communication channels listed below this newsletter. We look forward to seeing you there in person.
+
+## News from the community
+
+### Collaboration Cafes online call series
+
+Ever since it's launch in September 2019, already 3 online Collaboration Cafes have taken place on 4th, 18th and 19th September 2019. Thanks to all our attendees for coming along and joining the calls to learn more about these calls, tips on community management, GitHub tricks and the project itself. You can watch the recording from these calls on our [YouTube channel](https://www.youtube.com/channel/UCPDxZv5BMzAw0mPobCbMNuA).
+Read all about this online call series on [this page](https://github.com/alan-turing-institute/the-turing-way/blob/master/project_management/online-collaboration-cafe.md). We are still developing the format (see the [template](https://github.com/alan-turing-institute/the-turing-way/blob/master/communications/collaboration-cafe/collaboration-cafe-template.md)) and will be happy to get your feedback to make these sessions useful for you.
```

Comment:
```suggestion
Read all about this online call series on [this page](https://github.com/alan-turing-institute/the-turing-way/blob/master/project_management/online-collaboration-cafe.md).
We are still developing the format (see the [template](https://github.com/alan-turing-institute/the-turing-way/blob/master/communications/collaboration-cafe/collaboration-cafe-template.md)) and will be happy to get your feedback to make these sessions useful for you.
(For example, we've already added an "Australia-friendly" cafe on 30 October because Paula [got in touch](https://github.com/alan-turing-institute/the-turing-way/issues/684) 💖)
There's an issue open to collected your ideas ([#711](https://github.com/alan-turing-institute/the-turing-way/issues/711)).
Thank you in advance!
```